analytics/analytics.go

```diff
@@ -30,7 +30,8 @@ type Event struct {
 	Properties map[string]string `json:"properties,omitempty"`
 
 	DeviceID string `json:"device_id,omitempty"`
-	ServerID string `json:"server_id,omitempty"`
+	// TODO: is it safe to update the json name or do we need cloud coordination?
+	SessionID string `json:"server_id,omitempty"`
```

Comment:
This is a question for @vito @aluzzardi, not sure if we can just change the name of this json field and cloud will be fine or if it's more involved.

It doesn't *actually* matter that much, mostly aesthetics, but I guess discrepancies like this will add up in confusion in the long term.

Reply:
Looking at the server-side analytics code, I think you're good to rename this. There's a struct with this field defined, but nothing references it after unmarshaling. I can put up a quick PR to change it.

Reply:
Almost missed this, but awesome, just changed it to be session_id